WordPress/Sniffs/XSS/EscapeOutputSniff.php

```diff
@@ -152,6 +162,24 @@ public function process_token( $stackPtr ) {
 			if ( in_array( $function, array( 'trigger_error', 'user_error' ), true ) ) {
 				$end_of_statement = $this->phpcsFile->findEndOfStatement( $open_paren + 1 );
 			}
+		} else if ( false === $this->is_short_open_tag_enabled() && T_INLINE_HTML === $this->tokens[ $stackPtr ]['code'] ) {
+			// Skip if no PHP short_open_tag is in the string.
+			if ( false === strpos( $this->tokens[ $stackPtr ]['content'], '<?=' ) ) {
+				return;
+			}
+
+			// Report on what very likely is a PHP short open tag outputing variable
+			if ( preg_match( '/\<\?\=[\s\t]*(\$[a-zA-Z_][[:alnum:]_]+)[\s\t]*\;?[\s\t]*\?\>/', $this->tokens[ $stackPtr ]['content'], $matches ) ) {
```

Comment:
* The regex for a variable name in PHP is: `[a-zA-Z_\x7f-\xff][a-zA-Z0-9_\x7f-\xff]*` Ref: http://php.net/manual/en/language.variables.basics.php
* `'\t` is already included in `\s`, so no need to make it explicit in the character class.
* `;` does not need to be escaped in regex